### PR TITLE
Add simple /healthz handler in ctfe

### LINKED ISSUE #433: CTFE has no /healthz endpoint
Trillian has a /healthz endpoint that returns HTTP 200 OK and the string "ok" when it is ready to serve. The CTFE does not. It would be nice, for consistency at least, if it did. It appears that both "/" and "/metrics" are used in varying places as a substitute.

### PR DESCRIPTION
**Pull request by AlCutter**
Towards #433

### CODE REVIEW

File: trillian/ctfe/ct_server/main.go [hunk 1]
```diff
@@ -192,6 +192,12 @@ func main() {
 		}
 	})
 
+	// Export a healthz target.
+	corsMux.HandleFunc("/healthz", func(resp http.ResponseWriter, req *http.Request) {
+		//TODO(al): Wire this up to tell the truth.
```
Comment (pavelkalinnikov) [inline trillian/ctfe/ct_server/main.go]: Add whitespace after `//`

Comment (AlCutter) [reply trillian/ctfe/ct_server/main.go]: ta, done

File: trillian/ctfe/ct_server/main.go [hunk 2]
```diff
@@ -192,6 +192,12 @@ func main() {
 		}
 	})
 
+	// Export a healthz target.
+	corsMux.HandleFunc("/healthz", func(resp http.ResponseWriter, req *http.Request) {
+		// TODO(al): Wire this up to tell the truth.
```
Comment (RJPercival) [inline trillian/ctfe/ct_server/main.go]: Maybe create an Issue to track this, unless you're planning on doing it imminently?

Comment (AlCutter) [reply trillian/ctfe/ct_server/main.go]: I'll leave the current issue open and add a note there.

**Pull request merged**